### ISSUE TITLE
xlink:href attribute doesn't work

### ISSUE THREAD

**Issue by basham** [WebReflection/uhtml#10]
I'd like to dynamically set the `<use xlink:href>` attribute, so I can use SVG sprites easier. This is my ideal:

```js
const name = 'check'
const href = `#${name}` // Or `icons.svg#${name}`

render(target, html`
  <svg>
    <use xlink:href=${href}></use>
  </svg>
`)
```

However, uhtml doesn't seem to work well with the `xlink:href` attribute. Could you please review this, and determine whether this attribute should be included in some exception? Perhaps there's a different way to do this, that you'd recommend?

I put together a [demo on CodePen](https://codepen.io/basham/pen/gOpJxQX) with several examples to show how v1.9.0 works (1, 2, 4) and doesn't work (3, 5) with sprites. If it works, you should see a green check in the square box. I also commented out a line that would import the latest version of lighterhtml, so you can easily see that the same issue exists in that library. (I started experimenting with uce this week, so that's why I posted this issue to uhtml.)

Thanks for all your wonderful work and for your consideration!

**Comment by WebReflection** [comment]
The 3rd case is expected to not work, because you are using `html`, so that's not an _SVG_ node, it's just a useless _HTML_ one.

The 5th case seems to be the only one problematic to me, and I'll have a look ASAP, but 3 is a user error, not a library issue.

**Comment by basham** [comment]
I agree. Sorry for not clarifying. I provided Example 3 just to showcase specifically that it doesn't work (as expected). Example 5 doesn't work, and in contrast, I'd expect it to work. Example 5 is really what I'd like for you to explore more.

**Comment by basham** [comment]
I just added a new Example 6 (that doesn't work). It could be a compromise, if you need a stricter separation of HTML and SVG concerns. It essentially does this:

```js
const name = 'check'
const href = `#${name}` // Or `icons.svg#${name}`
const icon = svg`<use xlink:href=${href}></use>`

render(target, html`
  <svg>
    ${icon}
  </svg>
`)
```

**Comment by WebReflection** [comment]
as `xlink:href` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href), is there any reason you wouldn't use `href` instead? it works out of the box.

**Comment by basham** [comment]
Wow. Had no idea it was depreciated. Every example I've ever found used `xlink:href`, so I never bothered trying just `href`. But it works! So, yeah, I'm closing this as a non-issue. Thanks!

**Comment by WebReflection** [comment]
For reference sake, it looks like `href` is very well supported too:
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/href#Browser_compatibility
